packages/backend/src/open_payments/payment/incoming_remote/service.test.ts

```diff
@@ -85,69 +100,42 @@ describe('Remote Incoming Payment Service', (): void => {
           .mockResolvedValue(paymentPointer)
       })
 
-      test.each`
-        incomingAmount | expiresAt                        | description                | externalRef
-        ${undefined}   | ${undefined}                     | ${undefined}               | ${undefined}
-        ${amount}      | ${new Date(Date.now() + 30_000)} | ${'Test incoming payment'} | ${'#123'}
-      `('creates remote incoming payment ($#)', async (args): Promise<void> => {
-        const mockedIncomingPayment = mockIncomingPayment({
-          ...args,
-          paymentPointerUrl: paymentPointer.id
-        })
-
-        const grant = await grantService.create(grantOptions)
-
-        const clientCreateIncomingPaymentSpy = jest
-          .spyOn(openPaymentsClient.incomingPayment, 'create')
-          .mockResolvedValueOnce(mockedIncomingPayment)
-
-        const incomingPayment = await remoteIncomingPaymentService.create({
-          ...args,
-          paymentPointerUrl: paymentPointer.id
-        })
-
-        expect(incomingPayment).toStrictEqual(mockedIncomingPayment)
-        expect(clientCreateIncomingPaymentSpy).toHaveBeenCalledWith(
-          {
-            paymentPointer: paymentPointer.id,
-            accessToken: grant.accessToken
-          },
-          {
-            ...args,
-            expiresAt: args.expiresAt
-              ? args.expiresAt.toISOString()
-              : undefined,
-            incomingAmount: args.incomingAmount
-              ? serializeAmount(args.incomingAmount)
-              : undefined
-          }
-        )
-      })
-
-      test('returns error if grant expired', async () => {
+      test('returns error if grant expired and token cannot be rotated', async () => {
         await grantService.create({
           ...grantOptions,
           expiresIn: -10
         })
+        const clientRequestRotateTokenSpy = jest
+          .spyOn(openPaymentsClient.token, 'rotate')
+          .mockImplementationOnce(() => {
+            throw new Error('Error in rotating client')
+          })
 
         await expect(
           remoteIncomingPaymentService.create({
             paymentPointerUrl: paymentPointer.id
           })
         ).resolves.toEqual(RemoteIncomingPaymentError.ExpiredGrant)
+        expect(clientRequestRotateTokenSpy).toHaveBeenCalled()
       })
 
-      test('returns error if grant does not have accessToken', async () => {
+      test('returns error if rotated token does not have accessToken', async () => {
         await grantService.create({
           ...grantOptions,
-          accessToken: undefined
+          expiresIn: -10
         })
+        const clientRequestRotateTokenSpy = jest
+          .spyOn(openPaymentsClient.token, 'rotate')
+          .mockResolvedValueOnce({
+            access_token: { ...newToken.access_token, value: '' }
```

Comment:
I don't think this is possible.
https://github.com/interledger/open-payments/blob/f1b51c3c051c9030cea1f3d2542cfb492c4330d3/openapi/auth-server.yaml#L258-L261
https://github.com/interledger/open-payments/blob/f1b51c3c051c9030cea1f3d2542cfb492c4330d3/openapi/auth-server.yaml#L388-L406

Reply:
Then let's get rid of the test